Tests/SwiftSyntaxMacroExpansionTest/AccessorMacroTests.swift

```diff
@@ -44,6 +44,95 @@ fileprivate struct ConstantOneGetter: AccessorMacro {
 final class AccessorMacroTests: XCTestCase {
   private let indentationWidth: Trivia = .spaces(2)
 
+  func testAccessorOnVariableDeclWithTrailingLineCommentAndNoAccessorBlock() {
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int /*1*/ // hello
+      """,
+      expandedSource: """
+        var x: Int /*1*/  {// hello
```

Comment:
I wouldn't worry so much about only moving `//`/`///` style comments and instead just move all of the trailing trivia. That would also help with fixing the double space here (and we can just make sure to insert a space between `Int` and `{`.

Reply:
Thanks, I've made changes accordingly, please have a look.